src/utils/stopwatch.ts

```diff
@@ -1,18 +1,19 @@
 class Stopwatch {
 
-    private start:number = 0;
-    private end:number = 0;
+    private start: number = 0;
+    private end: number = 0;
 
     startWatch() {
-        this.start = Date.now();
+        this.start = new Date().getMilliseconds();
     }
 
     stopWatch() {
-        this.end = Date.now();
+        this.end = new Date().getMilliseconds();
     }
 
     getDuration(): number {
         return this.end - this.start;
     }
 }
-export { Stopwatch };
\ No newline at end of file
+
+export {Stopwatch};
```

Comment:
Please add a new line at the end of the file, GitHub is complaining. And it is a good practice in programming.

Reply:
ok